account_bank_statement_import_txt_xlsx/models/account_bank_statement_import_sheet_parser.py

```diff
@@ -204,15 +223,23 @@ def _parse_lines(self, mapping, data_file, currency_code):
                     timestamp,
                     mapping.timestamp_format
                 )
-
-            amount = self._parse_decimal(amount, mapping)
-            if balance is not None:
-                balance = self._parse_decimal(balance, mapping)
-
+            if amount_column:
+                amount = self._parse_decimal(
+                    values[amount_column], mapping)
             if debit_credit is not None:
                 amount = amount.copy_abs()
                 if debit_credit == mapping.debit_value:
                     amount = -amount
+            if debit_column and credit_column:
+                debit_amount = self._parse_decimal(
+                    values[debit_column], mapping)
+                debit_amount = debit_amount.copy_abs()
```

Comment:
Don't you mind adding a boolean field in the mapping telling if the debit column is always positive (true by default), so that you can easily handle both cases?

Reply:
My position is that if we have a way to find information, we don't ask for it. Because the parameters already contain several values.

Reply:
Yes sorry, both cases are already handled here, forget it.